test/elixir/test/changes_async_test.exs

```diff
@@ -103,6 +103,30 @@ defmodule ChangesAsyncTest do
     HTTPotion.stop_worker_process(worker_pid)
   end
 
+  @tag :with_db
+  test "eventsource no junk in response", context do
+    db_name = context[:db_name]
+
+    check_empty_db(db_name)
+    create_doc(db_name, sample_doc_foo())
+    create_doc_bar(db_name, "bar")
+    
```

Comment:
In the editor this line I think was showing as having 4 spaces. Ideally it would be just a newline (without trailing whitespace).